### ISSUE TITLE
Improve the message feature

### ISSUE THREAD

**Issue by shagun25** [shagun25/SheHeroes#9]
provide new ideas except following:
1.location
2. video and image capture(both back and front) 
after every 5 sec.

**Comment by subhangi2731** [comment]
can you please assign the issue to me?

**Comment by Sanju-github002** [comment]
I can do best for women safety so can u assing this problem to me.

**Comment by shagun25** [comment]
Both of you can drop your ideas in discord server of gssoc.

**Comment by aman-senpai** [comment]
> 
> 
> Both of you can drop your ideas in discord server of gssoc.

I'll love to suggest some ideas for this application, could you assist me with how to send them to you?

**Comment by shagun25** [comment]
> > Both of you can drop your ideas in discord server of gssoc.
> 
> I'll love to suggest some ideas for this application, could you assist me with how to send them to you?

through discord server

**Comment by kanchi12P** [comment]
please assign this issue to me I would like to give ideas for it.

**Comment by pranavi79** [comment]
This is my first time here, can I be assigned to this issue please

**Comment by NidhiChaurasia** [comment]
@shagun25 Could you please assign me this issue . So, that I could work on it

**Comment by shagun25** [comment]
@NidhiChaurasia share your ideas..please

**Comment by NidhiChaurasia** [comment]
> @NidhiChaurasia share your ideas..please

Yeah,Sure I would be telling you that.Pardon for the late reply cozI had my sem exams !

**Comment by NidhiChaurasia** [comment]
@shagun25 I have posted my idea on discord.Please Review and let me know about the chages I want to make ! Kindly Acknowledge......

**Comment by NidhiChaurasia** [comment]
@shagun25 We can enhance message feature by a Customized Chatbot service that automatizes the messaging feature based on the feed . My approach to #9
